harness/determined/tensorboard/base.py

```diff
@@ -53,3 +57,9 @@ def sync(self) -> None:
         Save the object to the backing persistent storage.
         """
         pass
+
+    def delete(self) -> None:
+        """
+        Delete all objects from the backing persistent storage.
+        """
+        pass
```

Comment:
If this method isn't optional (seems like it isn't), we should either raise a NotImplementedError or (better) we should do `class TensorboardManager(metaclass=abc.ABCmeta` and decorate `delete` with `@abc.abstractmethod`

Reply:
I'll make it an abstract class